conf_collectd_plugin.py

```diff
@@ -0,0 +1,375 @@
+        'The default is Off.\n'
+        'ExtendedStatus on')
+    out.close()
+
+
+# Apache Installation
+def apache_title():
+    sys.stdout.write(
+        " _____ __________  _____  _________   ___ ______________\n"
+        "   /  _  \\______   \/  _  \ \_   ___ \ /   |   \_   _____/\n"
+        "  /  /_\  \|     ___/  /_\  \/    \  \//    ~    \    __)_ \n"
+        " /    |    \    |  /    |    \     \___\    Y    /        \ \n"
+        " \____|__  /____|  \____|__  /\______  /\___|_  /_______  /\n"
+        "         \/                \/        \/       \/        \/ \n")
+
+
+def install_apache_plugin():
+    install = check_install_state("Apache")
+    print ""
+    if not install:
+        sys.stdout.write(
+            "This script has detected that you have apache installed and "
+            "running.")
+        res = utils.ask(
+            "Would you like to run the apache plugin installer to "
+            "enable collectd to collect data from apache?")
+    else:
+        print "You have previously installed this plugin."
+        res = utils.ask("Would you like to reinstall this plugin", no)
+
+    if not res:
+        return
+
+    apache_title()
+    print ""
+    sys.stdout.write(
+        "To enable collectd plugin with Apache, the following "
+        "steps need to be taken:\n"
+        "1. mod_status for apache needs to be enabled. (Default is enabled)\n"
+        "2. ExtendedStatus needs to be turned on.  (Default is off)\n"
+        "3. Enable the server-status handler for each virtual host.\n")
+
+    noop = raw_input("Press Enter to continue")
+    utils.print_step("Begin collectd Apache plugin installer")
+
+    """ check point
+    - check dependency
+    - change system file
+    - check mod status
+    - TODO: pull template
+    - prompt for user information
+    """
+    if not utils.command_exists("curl"):
+        utils.exit_with_failure("Curl is needed for this plugin.")
+
+    # ubuntu check
+    # Assumption:
+    # -latest apache2 is installed and the installation
+    # -directory is in the default place
+    utils.print_step("  Checking if mod_status is enabled")
+    cmd_res = utils.get_command_output("ls /etc/apache2/mods-enabled")
+    if "status.conf" not in cmd_res or "status.load" not in cmd_res:
+        utils.print_step("Enabling apache2 mod_status module.")
+        ret = utils.call_command("sudo a2enmod status")
+        if ret != 0:
+            utils.exit_with_message("a2enmod command was not found")
+    utils.print_success()
+    print ""
+    sys.stdout.write(
+        'In order to enable the apache plugin with collectd, the '
+        'ExtendedStatus setting must be turned on.\n'
+        'This setting can be turned on by having "ExtendedStatus on" '
+        'in one of the .conf file.\n')
+
+    sys.stdout.write(
+        'If you have already enabled this status, '
+        'answer "no" to the next question '
+        'and ignore the following warning.\n'
```

Comment:
it makes me a bit queasy to ask users to ignore warnings.  can you make the warning not show up somehow?